### PR TITLE
Removed Gem Release (replaced with Gemsmith).

### PR DESCRIPTION
**Pull request by bkuhlmann**
## Overview

[Gemsmith](https://github.com/bkuhlmann/gemsmith) automates the
generation of new gems adhering to many of the guidelines described in
this checklist. It also includes the setup of several tools (i.e. Git
Cop, Reek, Rubocop, etc.) not described in this checklist but worth of
including in any gem or Ruby project.

Gemsmith also provides improved support for publishing new gems
(including GPG signed versions -- highly recommended) and an auto-
generated change log which gets baked into each Git tag.
[Example](https://github.com/bkuhlmann/gemsmith/releases/tag/v10.3.0).

### CODE REVIEW

**Comment by bkuhlmann** [comment]
Doesn't look like this project is well maintained and want to keep my PR Inbox at zero.

**Comment by palkan** [comment]
Sometimes maintainers go on vacations and want to spend at least a dozen of days with Github notifications turned off.

Feel free to re-open the PR.

File: src/pages/en.md [hunk 1]
```diff
@@ -335,7 +335,7 @@ When should I bump a version?
 
 It's a good practice to publish RC/beta versions prior to releasing a major update.
 
-You can automate your release process by using, for example, [`gem-release`](https://github.com/svenfuchs/gem-release) or CI services (e.g. [Travis supports](https://docs.travis-ci.com/user/deployment/rubygems/) RubyGems deployments).
+You can automate your release process by using, for example, [`Gemsmith`](https://github.com/bkuhlmann/gemsmith) or CI services (e.g. [Travis supports](https://docs.travis-ci.com/user/deployment/rubygems/) RubyGems deployments).
```
Comment (palkan) [inline src/pages/en.md:338]: Let's not remove `gem-release` but mention both tools (i.e. "for example, Gemsmith or gem-release").

**Comment by palkan** [comment]
@bkuhlmann FYI https://github.com/palkan/gem-check/commit/61abedcdcaa6cca1883a5210c023e1a76062c479

**Comment by bkuhlmann** [comment]
Thanks, that'll be nice. :)

**Pull request closed without merging**